scripts/api-inference/scripts/generate.ts

```diff
@@ -373,15 +377,18 @@ await Promise.all(
           description: string;
           inference: string | undefined;
           config: JsonObject | undefined;
+          tags: string[];
         }) => {
           console.log(`   ⚡ Checking inference status ${model.id}`);
           let url = `https://huggingface.co/api/models/${model.id}?expand[]=inference`;
           if (task === "text-generation") {
             url += "&expand[]=config";
           }
+          url += "&expand[]=tags";
```

Comment:
I don't think we should fetch the tags for the recommended model here. The only place it's used is to check for `"conversational"` entry. However, it's best to hard-code the `"conversational"` tag when we know we want to display the chat completion snippet and remove it when we want to display the text generation snippet.

A same model can be used either for text-generation (with client-side rendered prompt, as in llama-stack) or for chat-completion (with server-side rendered prompt, as everywhere else). In the documentation, we have 2 different pages [Text Generation](https://huggingface.co/docs/api-inference/tasks/text-generation) and [Chat Completion](https://huggingface.co/docs/api-inference/tasks/chat-completion). We should pass the `"conversational"` tag only for the second page, regardless of which model is recommended.